LegitimateURLShortener.txt

```diff
@@ -1095,8 +1094,16 @@ $removeparam=tg_ref
 $removeparam=dt_dapp
 $removeparam=dt_platform
 
+! https://gitlab.com/anti-tracking/ClearURLs/rules/-/issues/200
+! https://tenshoku.nikkei.co.jp/lp/15002/?n_cid=TPRN0002&waad=Ujim9yQ8&adid=NK_L2_T_shortlink_Sun_diagnosis&utm_source=NKdenshiban&utm_medium=content-text&utm_content=text&utm_campaign=shortlink_Sun_diagnosis
+! https://xtech.nikkei.com/atcl/nxt/column/18/01778/00009/?i_cid=nbpnxt_ranking
+! https://gooday.nikkei.co.jp/atcl/column/21/101900007/102000001/?waad=KOqJ8ACd
+$removeparam=adid
```

Comment:
`adid` is already being removed here: https://github.com/DandelionSprout/adfilt/blob/78e91db13d908b7a10ae6edb4dafbb910f8287ed/LegitimateURLShortener.txt#L375

Reply:
Good point

Reply:
Oh, yes, I have removed it. Thanks!